.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+### Checklist
+
+- [ ] Update changelog in CHANGELOG.md.
+- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
```

Comment:
I know this is taken from elsewhere but I don't actually know what to do here -- do you know? If so, can you link or describe?

Reply:
I removed it - `values.schema.json` is an optional validation file which you can use to ensure `values.json` is structured correctly.

We don't use it in this repo - therefor removing it :)